packages/fxa-auth-server/lib/senders/emails/renderer.ts

```diff
@@ -6,37 +6,18 @@ import ejs = require('ejs');
 import mjml2html = require('mjml');
 import * as templates from './templates';
 import * as layouts from './layouts';
-import path = require('path');
-const config = require('../../../config').getProperties();
 
 const mjmlConfig: Record<any, any> = {
-  validationLevel: 'soft',
-  // ignoreIncludes: config.env === 'test' ? true : false,
-};
-
-export const context = {
-  buttonText: 'Sync another device',
-  onDesktopOrTabletDevice: true,
-  anotherDeviceUrl:
-    config.contentServer.url +
-    '/connect_another_device?utm_medium=email&utm_campaign=fx-cad-reminder-first&utm_content=fx-connect-device',
-  preHeader: 'random headers',
-  privacyUrl: config.smtp.privacyUrl,
-  supportUrl: config.smtp.supportUrl,
-  oneClickLink: true,
-  iosUrl:
-    'https://accounts-static.cdn.mozilla.net/product-icons/apple-app-store.png',
-  androidUrl:
-    'https://accounts-static.cdn.mozilla.net/product-icons/google-play.png',
-  subject: 'Reminder to sync your device',
-  cssBaseDir: `${__dirname}/css`,
+  validationLevel: 'strict',
 };
 
 function compile(
   context: Record<any, any>,
   templateName: string,
   subTemplate?: string
 ) {
+  const ignoreIncludes = typeof global.it === 'function';
```

Comment:
The CI issue seems to have been resolved with this addition. What I'm doing is detecting if mocha test is running the file and accordingly setting this config parameter to ignore `<mj-include>`s. Not sure if this is the best way but let me know your thoughts on this!

Reply:
This is definitely clever, and for now I think it'll be fine, but I would recommend leaving a comment in the code to help others understand why it was neccesary (you could even put the PR # in there)

Reply:
+1 on leaving a comment